sdk/digitaltwins/digital-twins-core/review/digital-twins-core.api.md

```diff
@@ -41,11 +41,35 @@ export interface DigitalTwinsAddHeaders {
     etag?: string;
 }
 
+// @public
+export interface DigitalTwinsAddOptionalParams extends coreHttp.OperationOptions {
+    digitalTwinsAddOptions?: DigitalTwinsAddOptions;
```

Comment:
So this is used like this?

```ts
await client.upsertDigitalTwin(twinId, twinJson, { digitalTwinsAddOptions: { ifNoneMatch }});
```

Why is it prefixed with `digitalTwinsAddOptions`? Usually things that extend `operationOptions` don't nest custom options like this. 

Why not have this?

```ts
await client.upsertDigitalTwin(twinId, twinJson,  { ifNoneMatch });
```

/cc @bterlson

Reply:
I don't want to go force and back with this. Brian told me I should hide the ifNoneMatch in the option bag. (Python also doing the same. What you are asking for is the same what we have yesterday..

Reply:
Fixed

Reply:
resolved offline: we found out this strange grouping was because of `x-ms-parameter-grouping` being used in the swagger. Removing this directive with a transform fixed the issue.